specification/reservations/resource-manager/Microsoft.Capacity/preview/2020-10-01-preview/reservations.json

```diff
@@ -1482,6 +1525,59 @@
         }
       }
     },
+    "ChangeDirectoryResponse": {
+      "type": "object",
+      "description": "Change direcotry response",
+      "properties": {
+        "reservationOrder": {
+          "type": "object",
+          "description": "Change direcotry result for reservation order",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "Identifier of the reservation order"
+            },
+            "name": {
+              "type": "string",
+              "description": "Name of the reservation order"
+            },
+            "isSucceeded": {
+              "description": "True if change directory operation succeeded on this reservation order",
+              "type": "boolean"
+            },
+            "error": {
+              "type": "string",
+              "description": "Error reason if operation failed. Null otherwise"
+            }
+          }
+        },
+        "reservations": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "description": "Change direcotry result for reservation",
+            "properties": {
+              "id": {
```

Comment:
if the schema of the reservations result and reservationorder result is meant to be the same, consider a separate definition for it and ref'ing it from these 2

Reply:
will use ref